pwiz_tools/Skyline/TestRunnerLib/RunTests.cs

```diff
@@ -212,6 +222,8 @@ private void SetTestDir(TestContext testContext, string resultsDir)
         {
             if (string.IsNullOrEmpty(resultsDir))
                 resultsDir = Path.Combine(GetProjectPath("TestResults"), "TestRunner results");
+            else if (IsParallelClient && resultsDir.Contains("TestResults_"))
```

Comment:
Still not seeing the value.

Reply:
He's using the TestResults_n directory to get the worker id. Seems potentially useful to me (especially using it as he is to say which worker is producing the error output) and I'm not seeing a way to get it from environment variables inside the workers. Do you think it would be better to have a dedicated TestRunner parameter for the worker name?

Reply:
Seems like a kludge to me to have a useful value appended to a folder name where it is not necessary and then parse the path to get the value. Definitely having an explicit TestRunner argument for the value seems like a better route to me.

Reply:
I thought about using another parameter there, yeah. Certainly would want to do that if the info wasn't otherwise encoded, but I do like the richer information in the log from having it in the test directory name so it just seemed like extra work.

Reply:
Okay. Just add some comments explaining why you did what you did and that it is not actually necessary to keep tests from using the same files.